packages/admin/src/error/errorboundary/useErrorBoundary.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+const useErrorBoundary = () => {
+    const [, setState] = React.useState(false);
+    const throwException = React.useCallback((e: Error) => {
+        setState(() => {
+            throw e;
+        });
+    }, []);
+    return { throwException };
```

Comment:
`useAsyncError` instead of `useErrorBoundary`?
`throwError` instead of `throwException`?

And maybe even return directly the callback so that it can be used like this:
```
const throwAsyncError = useAsyncError();
throwAsyncError (new Error("Async error"));
```

Reply:
`useErrorBoundary` is ok imho

throwError or just throw? or does that conflict with the throw keyword?

Reply:
if we add this hook, i think it should be possible to rename the method to throw